tools/static-data/generate-azure-pipelines-tasks.mjs

```diff
@@ -49,12 +49,15 @@ await (async () => {
       try {
         // Get the content of the file at the commit
         const content = await git.show([`${rev}:${file}`]);
-        /** @type {{name: string, version: {Major: number, Minor: number, Patch: number}}} */
+        /** @type {{name: string, id: string, version: {Major: number, Minor: number, Patch: number}}} */
         const parsedContent = JSON5.parse(content);
         const version = `${parsedContent.version.Major}.${parsedContent.version.Minor}.${parsedContent.version.Patch}`;
         tasks[parsedContent.name.toLowerCase()] =
           tasks[parsedContent.name.toLowerCase()]?.add(version) ??
           new Set([version]);
+        tasks[parsedContent.id.toLowerCase()] =
```

Comment:
are you sure we need the to lower case here?

Reply:
Yes unfortunately. The ID is a GUID and can be in any case. The data file has been normalized to lowercase.